SQLQuery_3(homework).sql

```diff
@@ -0,0 +1,303 @@
+CREATE DATABASE ZooManagement;
+
+USE ZooManagement;
+
+CREATE TABLE ZooInstitutions (
+    Id INT IDENTITY(1,1) PRIMARY KEY,
+    Name VARCHAR(100) NOT NULL, 
+    City VARCHAR(150) DEFAULT 'Country Romania',
+);
+
+
+CREATE TABLE Visitors (
+    Id INT IDENTITY(1,1) PRIMARY KEY,
+    TypeOfDay VARCHAR(50) NOT NULL,
```

Comment:
For TypeOfDay a new table, maybe called, WeekDays could be used. Inside WeekDays we can store information like "Monday", "Tuesday" etc. and then it can be referenced in the Tickets table.